tests/files/dummy_data.sql

```diff
@@ -27,10 +27,20 @@ INSERT INTO loaders (id, loader) VALUES (1, 'fabric');
 INSERT INTO loaders_project_types (joining_loader_id, joining_project_type_id) VALUES (1,1);
 INSERT INTO loaders_project_types (joining_loader_id, joining_project_type_id) VALUES (1,2); 
 
-INSERT INTO categories (id, category, project_type) VALUES (1, 'combat', 1);
-INSERT INTO categories (id, category, project_type) VALUES (2, 'decoration', 1);
-INSERT INTO categories (id, category, project_type) VALUES (3, 'economy', 1);
+INSERT INTO categories (id, category, project_type) VALUES
+    (1, 'combat', 1),
+    (2, 'decoration', 1),
+    (3, 'economy', 1),
+    (4, 'food', 1),
+    (5, 'magic', 1),
+    (6, 'mobs', 1),
+    (7, 'optimization', 1);
 
-INSERT INTO categories (id, category, project_type) VALUES (4, 'combat', 2);
-INSERT INTO categories (id, category, project_type) VALUES (5, 'decoration', 2);
-INSERT INTO categories (id, category, project_type) VALUES (6, 'economy', 2);
\ No newline at end of file
+INSERT INTO categories (id, category, project_type) VALUES
+    (8, 'combat', 2),
```

Comment:
if we forsee adding more testing categories in the future it might be worth starting this second set at 10 or 20. If you think this is enough, should be OK

Reply:
Good suggestion, I made the next ones start at 101